source/includes/openshift/deploy-minio-on-openshift.rst

```diff
@@ -16,7 +17,7 @@ Overview
 Red Hat® OpenShift® is an enterprise-ready Kubernetes container platform with full-stack automated operations to manage hybrid cloud, multi-cloud, and edge deployments. 
 OpenShift includes an enterprise-grade Linux operating system, container runtime, networking, monitoring, registry, and authentication and authorization solutions. 
 
-You can deploy the MinIO Kubernetes Operator through the :openshift-docs:`Red Hat® OpenShift® Container Platform 4.7+ <welcome/index.html>`. 
+You can deploy the MinIO Kubernetes Operator through the :openshift-docs:`Red Hat® OpenShift® Container Platform 4.8+ <welcome/index.html>`. 
```

Comment:
Per Celis, Red Hat said we have to be `4.8+` now.

Reply:
@cniackz is there a specific Operator version where that bump happened? We might want to maintain a matrix somewhere, at least internally.

Reply:
Hello @ravindk89,

Our ability to influence this situation is limited. I recall that certification tests would fail if we attempted to cover versions 4.7 or older, necessitating our decision to cease coverage of those versions and progress towards more recent ones.

Furthermore, if you visit their website, you will notice that full support for that particular version has already concluded: https://access.redhat.com/support/policy/updates/openshift

Best regards,

<img width="1170" alt="Screenshot 2023-09-25 at 11 09 48 AM" src="https://github.com/minio/docs/assets/6667358/dcd6a047-82ec-4cfa-9fbb-a31911d2e577">